lib/systems/parse.nix

```diff
@@ -335,6 +340,8 @@ rec {
   mkSkeletonFromList = l: {
     "1" = if elemAt l 0 == "avr"
       then { cpu = elemAt l 0; kernel = "none"; abi = "unknown"; }
+      else if elemAt l 0 == "wasm32" || elemAt l 0 == "wasm64"
+        then { cpu = elemAt l 0; vendor = "unknown"; kernel = "wasm"; abi = "wasm"; }
```

Comment:
The "wasm" kernel is probably incorrect... Not sure how else to get it work though